addons/purchase_stock/models/stock.py

```diff
@@ -163,7 +163,7 @@ class Orderpoint(models.Model):
 
     def _quantity_in_progress(self):
         res = super(Orderpoint, self)._quantity_in_progress()
-        for poline in self.env['purchase.order.line'].search([('state','in',('draft','sent','to approve')),('orderpoint_id','in',self.ids)]):
+        for poline in self.env['purchase.order.line'].search([('state', 'in', ('draft', 'sent', 'to approve')), ('orderpoint_id', 'in', self.ids), ('move_dest_ids', '=', False)]):
```

Comment:
i would add in the commit message "now that the we propagate the orderpoint_id, it's important to ignore po line with an orderpoint AND a chained stock move, else the quantities will be counted twice by the orderpoint in a 2 steps receipt scenario"